pkg/container/id/id.go

```diff
@@ -18,6 +18,9 @@ func NewFromV2(idV2 *refs.ContainerID) *ID {
 }
 
 // New creates and initializes blank ID.
+//
+// Defaults:
+//  - value: "".
```

Comment:
We still doc that `value: nil` since this type is a reflection of API message, and everyone will understand it. But type don't provide getter of the value, so we just don't need to cover it in unit test and test only `ToV2`. Stringer is also implemented, but if you want to underline that `(*ID)(nil).String() == ""` then it's better to do it in `String` method's doc and cover in dedicated test.

Reply:
changed default value to `nil` in comment